### PR TITLE
Users/shrer/single ci for vs test console flow

### PR DESCRIPTION
**Pull request by ShreyasRmsft**
Consolidated ci event for the vstest.console flow. Bug fix in the test code where the path to chcp.com had to be windows\.... with a small w.

### CODE REVIEW

File: Tasks/VsTest/distributedtest.ts [hunk 1]
```diff
@@ -147,17 +147,20 @@ export class DistributedTest {
 
         var consolidatedCiData = {
                     agentFailure: false,
+                    agentPhaseSettings: tl.getVariable('System.ParallelExecutionType'),
```
Comment (cltshivash) [inline Tasks/VsTest/distributedtest.ts]: confirm this variable is the used across build and release with Nitin ?

File: Tasks/VsTest/vstest.ts [hunk 2]
```diff
@@ -60,14 +60,32 @@ export function startTest() {
...
+            result: 'Failed',
+            settingsType:  !utils.Helper.isNullOrUndefined(vstestConfig.settingsFile) ? vstestConfig.settingsFile.endsWith('.runsettings') ? 'runsettings' : vstestConfig.settingsFile.endsWith('.testsettings') ? 'testsettings' : 'none': 'none',
+            testSelection: vstestConfig.testSelection,
+            tiaEnabled: vstestConfig.tiaConfig.tiaEnabled,
+            vsTestVersion: vstestConfig.vsTestVersionDetails.majorVersion + '.' + vstestConfig.vsTestVersionDetails.minorversion + '.' + vstestConfig.vsTestVersionDetails.patchNumber
+        };
+
         invokeVSTest().then(function (taskResult) {
             uploadVstestDiagFile();
             if (taskResult == tl.TaskResult.Failed) {
```
Comment (cltshivash) [inline Tasks/VsTest/vstest.ts]: do we have partially succeeded ?

File: Tasks/VsTest/distributedtest.ts (hunk 1 above)
Comment (ShreyasRmsft) [reply Tasks/VsTest/distributedtest.ts]: Okay. Will check

File: Tasks/VsTest/vstest.ts (hunk 2 above)
Comment (ShreyasRmsft) [reply Tasks/VsTest/vstest.ts]: Nope

**Pull request merged**